src/app/components/named-entity/named-entity-occurrence/named-entity-occurrence.component.ts

```diff
@@ -8,4 +12,20 @@ import { NamedEntityOccurrence } from '../../../models/evt-models';
 })
 export class NamedEntityOccurrenceComponent {
   @Input() occurrence: NamedEntityOccurrence;
+  @Input() entityId: string;
+
+  constructor(
+    private evtModelService: EVTModelService,
+    private evtStatusService: EVTStatusService,
+  ) {
+  }
+
+  goToOccurrenceRef(ref: NamedEntityOccurrenceRef) {
+    this.evtModelService.pages$.pipe(first()).subscribe(pages => {
```

Comment:
If you are not interested in the error notification or in the predicate function, use `take(1)` instead of `first()`